### ISSUE TITLE
More training detail

### ISSUE THREAD

**Issue by Zyq-scut** [neuralchen/SimSwap#56]

**Comment by Zyq-scut** [comment]
Hi, thank for this incredible work. I am trying to re-implement the training code, but i can't find enough training detail in paper. Can you please provide more detail such as learning rate, scheduler and any augmentation of data? And what gpu do you use to train? i train on 100 thousand pictures with 4 2080ti gpu and batch size 12, which take me 3 days but only go through 25 epoch. This looks like it will take a long time to train 500 epoch. Do you have any advise on imporve training speed? Thanks again.

**Comment by GianniGi** [comment]
more scripts are coming, from the home page:
"Currently, only the test code is available. Training scripts are coming soon"

**Comment by neuralchen** [comment]
Training scripts are available now!
